src/components/RenderRouter/HeroItem.tsx

```diff
@@ -247,6 +247,7 @@ class HeroItem extends React.Component<Props, State> {
       this.downArrowScroll();
     };
     if (this.state.content.style === 'full') {
+      console.log(moment().weekday());
```

Comment:
I assume this was for testing/debugging. Perhaps remove?

Reply:
Whoops 😅